app/models/appeal_submission.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class AppealSubmission < ApplicationRecord
+  self.ignored_columns = ['board_review_otpion']
```

Comment:
I know dangerbot is going to complain about this line but `strong_migrations` insisted on it.  So, botfight! 🤖 💥